components/HeroSectionLayout/HeroSectionLayout.tsx

```diff
@@ -10,38 +12,95 @@ type HeroSectionProps = {
                 <Stack gap={8}>
                     <Flex flexDir="column">
-                        {identifier && <Text variant="brandHighlight">{identifier}</Text>}
-
-                        {heading && (
-                            <Heading fontSize={['2em', '2em', '3em', '3em']} fontWeight="600" color="black">
-                                {heading}
-                            </Heading>
+                        {identifier && (
+                            <Animation index={0}>
+                                <Text variant="brandHighlight">{identifier}</Text>
+                            </Animation>
                         )}
 
-                        {customHeading}
+                        <Animation index={1}>
+                            {heading && (
+                                <Heading fontSize={['2em', '2em', '3em', '3em']} fontWeight="600" color="black">
+                                    {heading}
+                                </Heading>
+                            )}
+                        </Animation>
+
+                        {/* Hack for the front page to set blend mode here */}
+                        <Animation style={{ mixBlendMode: 'overlay' }}>{customHeading}</Animation>
                     </Flex>
 
                     {paragraph && (
-                        <Text lineHeight="2" color="black" maxWidth={'750px'}>
-                            {paragraph}
-                        </Text>
+                        <Animation index={2}>
+                            <Text lineHeight="2" color="black" maxWidth={'750px'}>
+                                {paragraph}
+                            </Text>
+                        </Animation>
                     )}
 
-                    {customParagraph}
+                    {customParagraph && <Animation index={2}>{customParagraph}</Animation>}
 
-                    <ContactButton />
+                    <Animation index={3}>
+                        <ContactButton />
+                    </Animation>
                 </Stack>
             </SectionItem>
-            <SectionItem colStart={[1, 1, 4]} colEnd={[8, 8, 6, 6]}>
-                {content}
+
+            <SectionItem colStart={[1, 1, 4]} colEnd={[8, 8, 6, 6]} zIndex={200}>
+                <motion.div
+                    initial={{ scale: 0.8, opacity: 0 }}
+                    animate={{
+                        scale: 1,
+                        opacity: 1,
+                    }}
+                    transition={{ duration: 1, ease }}>
+                    {shake && (
+                        <motion.div
+                            whileHover={{
+                                rotate: ['0deg', '6deg', '-8deg', '10deg', '-2deg', '0deg'],
+                                transition: { duration: 0.4 },
+                            }}>
+                            {content}
+                        </motion.div>
+                    )}
+                    {!shake && <>{content}</>}
```

Comment:
Det er en sød lille shake animation du har lavet!

Reply:
Måske vi kan bruge den et sted hvor vi har brug for at brugeren bliver opmærksom på noget. 👍